source/ingress.go

```diff
@@ -65,6 +73,28 @@ func endpointsFromIngress(ing *v1beta1.Ingress) []*endpoint.Endpoint {
 		return endpoints
 	}
 
+	if len(ing.Spec.Rules) == 0 && fqdntemplate != "" {
+		tmpl, err := template.New("endpoint").Funcs(template.FuncMap{
```

Comment:
does it really make sense to apply template for ingress resources ? I would expect host to be defined for each ingress object and if not, it should be skipped.